newrelic/resource_newrelic_synthetics_secure_credential_test.go

```diff
@@ -10,17 +10,18 @@ import (
 	"testing"
 	"time"
 
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 	"github.com/newrelic/newrelic-client-go/v2/pkg/entities"
 )
 
 func TestAccNewRelicSyntheticsSecureCredential_Basic(t *testing.T) {
 	resourceName := "newrelic_synthetics_secure_credential.foo"
-	rName := generateNameForIntegrationTestResource()
+	rName := acctest.RandString(7)
 
```

Comment:
wouldn't this be excluded during cleanups?

Reply:
Yes, good call. I'll update.